src/pure-functions/update.ts

```diff
@@ -67,3 +67,14 @@ const objectDeepDiff = (objectA: any, objectB: any) => {
   }
   return true
 }
+
+export const checkUntilChanged = async (gapMilliseconds: number, maxRetry: number, judgement: () => Promise<boolean> | boolean): Promise<boolean> => {
```

Comment:
It's better to use `PUPPET_PAYLOAD_SYNC_MAX_RETRY` as default value for `maxRetry`.

Reply:
I think it's better to use the default value outside the call because we might use this function somewhere else in the future and it may have a different default value.